schema.sql

```diff
@@ -30,4 +30,38 @@ ADD COLUMN owner_id integer,
 ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id) ON DELETE CASCADE,
 ADD CONSTRAINT fk_owner FOREIGN KEY(owner_id) REFERENCES owners(id) ON DELETE CASCADE;
 
+CREATE TABLE vets (
+  id integer PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(255),
+  age integer,
+  date_of_graduation date
+);
+
+CREATE TABLE specializations (
+  species_id integer,
+  vet_id integer,
+  CONSTRAINT species_fk FOREIGN KEY (species_id) REFERENCES species(id) ON DELETE CASCADE,
+  CONSTRAINT vets_fk FOREIGN KEY (vet_id) REFERENCES vets(id) ON DELETE CASCADE
+);
+
+CREATE TABLE visits (
+    animal_id integer,
+    vet_id integer,
+    visit_date date,
+    CONSTRAINT animals_fk FOREIGN KEY (animal_id) REFERENCES animals(id) ON DELETE CASCADE,
+    CONSTRAINT vets_fk FOREIGN KEY (vet_id) REFERENCES vets(id) ON DELETE CASCADE
+);
 
+SELECT species.name AS suggested_specialty, COUNT(visits.animal_id) AS visit_count
+FROM species
+INNER JOIN specializations ON species.id = specializations.species_id
+INNER JOIN vets ON specializations.vet_id = vets.id
+INNER JOIN visits ON specializations.vet_id = visits.vet_id AND visits.animal_id IN (
+    SELECT animal_id
+    FROM visits
+    INNER JOIN vets ON visits.vet_id = vets.id
+    WHERE vets.name = 'Maisy Smith'
+)
+GROUP BY species.id, species.name
+ORDER BY visit_count DESC
+LIMIT 1;
```

Comment:
- [ ] Great job so far @marvin-nyalik There is a small change I will suggest you implement and you'll be on your way to the next Milestone :+1: 
- [ ] Per the project requirements, you are supposed to put correct `SQL` commands in each of the files.
- Everything that creates or changes the structure of the table should be saved in the `schema.sql` file :+1: 
- Everything that populates tables with data should be saved in the data.sql file :+1: 
- Everything that is a query that helps to analyze data should be saved in the `queries.sql` file :+1: 
- [ ] Currently, this command is a query and as a result needs to be saved in the `queries.sql` file. So kindly fix that :+1: